specification/apicenter/data-plane/ApiCenter.DataApi/preview/2024-02-01-preview/azure.apicenter.dataapi.json

```diff
@@ -0,0 +1,1644 @@
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Azure.Core.Foundations.Error",
+          "description": "The error object."
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Azure.Core.Foundations.InnerError": {
+      "type": "object",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "Inner error."
+        }
+      }
+    },
+    "Azure.Core.Foundations.OperationState": {
+      "type": "string",
+      "description": "Enum describing allowed operation states.",
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "OperationState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "NotStarted",
+            "value": "NotStarted",
+            "description": "The operation has not started."
+          },
+          {
+            "name": "Running",
+            "value": "Running",
+            "description": "The operation is in progress."
+          },
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "The operation has completed successfully."
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "The operation has failed."
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "The operation has been canceled by the user."
+          }
+        ]
+      }
+    },
+    "Models.Api": {
+      "type": "object",
+      "description": "API resource model.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The resource name.",
+          "maxLength": 50,
+          "readOnly": true
+        },
+        "title": {
+          "type": "string",
+          "description": "The name of the API.",
+          "maxLength": 50
+        },
+        "summary": {
+          "type": "string",
+          "description": "Short description of the API.",
+          "maxLength": 200
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the API",
+          "maxLength": 1000
+        },
+        "kind": {
+          "$ref": "#/definitions/Models.ApiKind",
+          "description": "Kind of API. For example, REST or GraphQL."
+        },
+        "lifecycleStage": {
```

Comment:
The `lifecycleStage` (in API entity specifically) should be marked `readOnly`. It is being calculated from API version states.